### ISSUE TITLE
Koala 2.2 won't start, Win 7

### ISSUE THREAD

**Issue by mhenry1378** [oklai/koala#655]
**Windows 7, 64-bit, 16GB RAM, Service Pack 1**

**Bug Report**
Downloaded Koala 2.2 today (6/15/2017), updating from an earlier version that worked fine.
App will no longer load.

**Similar to:**
https://github.com/oklai/koala/issues/630

**Comment by zaygraveyard** [comment]
What was your previous version?

**Comment by mhenry1378** [comment]
Sorry, I don't honestly know and I've deleted all previous exe files.

**Comment by zaygraveyard** [comment]
How long has it been approximately since the last time you updated? more than 3 months?
Thanks

**Comment by mhenry1378** [comment]
Yes, probably more than six months. Been using it for close to two years. Can't recall if I ever updated it.

**Comment by zaygraveyard** [comment]
Then it was probably v2.0.4

**Comment by zaygraveyard** [comment]
BTW you can revert back to the [previous version](https://github.com/oklai/koala/releases/tag/v2.0.4), while I try to fix it.

**Comment by mhenry1378** [comment]
Great. Thank you. Looking forward to the fix.
